### ISSUE TITLE
The frame not change after set currentTime

### ISSUE THREAD

**Issue by Statfine** [bbc/VideoContext#67]
the frame is not change when VideoContext is paused  and set currentTime。。。
like this, the VideoContext is pause in 5s， the frame is stop in 5s，then change the currentTime to 10s but the frame is still in 5s,
Is that right? Or what do I need to do to change the picture?

**Comment by gossi** [comment]
I have the same problem

**Comment by PTaylour** [comment]
Hi guys, thanks for reporting this. I'll look to revert the v0.51.1 change.

@MatthewShotton did https://github.com/bbc/VideoContext/commit/617528f78de2006084803a4fa54d6672f70b4095 close another issue? Be good to avoid reimplementing a different problem.

**Comment by MatthewShotton** [comment]
@PTaylour I've had a play around with reverting that change and it doesn't seem to reintroduce an old bug (although i'm not 100% sure on what bug that change was meant to fix, or why reverting it fixes this issue). I think it's probably safe to roll it back for the time being.

**Comment by MatthewShotton** [comment]
We're still trying to get to the bottom of this. It looks like some low level behaviors vary between browser manufacturers.

Testing using this sample put together by @PTaylour 

```HTML
<!DOCTYPE html>
<html>
<head>
	<script type="text/javascript" src="../dist/videocontext.js"></script>
<body>
	<script type="text/javascript">
		window.onload = function(){
		var canvas = document.getElementById("canvas");
		window.vc = new VideoContext(canvas);
		var videoNode = vc.video ("../docs/assets/introductions-rant.mp4", 10);
		videoNode.connect(vc.destination);
		videoNode.start(0);
		vc.play ();
		setTimeout(() => { vc.pause() }, 1000);
		setTimeout(() => { vc.currentTime = 15 }, 2000);
		}
	</script>
    <p>
        <canvas id="canvas" width="480", height="270"></canvas>
    </p>

</body>
</html>
```

Replicates the issue using the current version of the library on Firefox and Chrome. Firefox gives us the error: 

`Error: WebGL warning: drawArrays: Active texture 0 for target 0x0de1 is 'incomplete', and will be rendered as RGBA(0,0,0,1), as per the GLES 2.0.24 $3.8.2: The dimensions of `level_base` are not all positive.`

Which hints that an unloaded or partially loaded image is trying to be uploaded as a texture and it's causing it to be displayed as black instead.

By rolling back 617528f we'd add an extra check to make sure the readyState of the underlying video is at a point where it has valid image data to upload.....in theory at least. Rolling back this change only seems to work on Firefox, not Chrome. There's also the added question of why, when the underlying element does load there's still a broken texture until play is called.

**Comment by gossi** [comment]
I do have this problem in chrome, well electron to be fair (electron `v1.7.9`), which uses an older chrome version.

**Comment by PTaylour** [comment]
We've published 0.51.6 with the changed rolled back. Thanks for your input guys, let me know how you get on!

**Comment by Statfine** [comment]
yes， is OK now
